contracts/rust/adapter/src/sol_types.rs

```diff
@@ -373,38 +419,20 @@ impl<'de> Deserialize<'de> for ValidatorRegisteredV2 {
     }
 }
 
-impl Serialize for CommissionUpdated {
```

Comment:
We're assuming there are no `CommissionUpdated` events in storage right? It's a bit risky because it's currently not known when this change will be deployed. So we do need to make sure we don't deploy `main` from before this change after we have upgraded the stake table contract and hope that no other validator does that either.

Reply:
I don't think this is likely to happen but it's still some extra risk and another thing to keep track of.

Reply:
yeah I don't think we are deploying the commission changes on decaf, so it should be fine IMO

Reply:
I think the `alloy v1.0` will derive serde by default, we don't need those manual derivation?

Reply:
We need it for migrating existing storage.